catroid/src/main/java/org/catrobat/catroid/CatroidApplication.java

```diff
@@ -41,6 +48,8 @@
 	public static final String OS_ARCH = System.getProperty("os.arch");
 
 	public static boolean parrotLibrariesLoaded = false;
+	public static String defaultSystemLanguage;
```

Comment:
This variable is not really necessary as it can already be retreived from a static context.